src/vue/text-editor/actions-menu-text-editor.vue

```diff
@@ -1,14 +1,20 @@
 <template>
     <div class="alpheios-alignment-editor-actions-menu">
       <div class="alpheios-alignment-editor-actions-menu__buttons">
+        <!--
         <button class="alpheios-editor-button-tertiary alpheios-actions-menu-button"  id="alpheios-actions-menu-button__download"
             @click="downloadSingle" :disabled="!downloadAvailable" >
             {{ l10n.getMsgS('ACTIONS_DOWNLOAD_TITLE') }}
         </button>
+        -->

```

Comment:
I commented for now - because there is not fully clear would this button be used.